src/main/resources/db/migration/V6_1__update_behandler_herid.sql

```diff
@@ -0,0 +1 @@
+UPDATE BEHANDLER SET her_id='180457',updated_at=now() WHERE ID=32483 AND kontor_id=978;
```

Comment:
Som det ble kommentert på en annen PR så er jo "AND kontor_id=978" overflødig her, men gjør ikke noe å være eksplitt på kontoret også (dessuten så kan jo id=32483 referere til noe helt annet i test-databasen).

Reply:
Du får med deg litt av hvert selv om du er på ferie, altså ;)